### DISCUSSION TITLE
Codeigniter 3

### DISCUSSION THREAD

**Discussion by rfulcher** [peachpiecompiler/peachpie#1116] [Q&A]
I have a project in Codeigniter 3 that I want to start porting over to .Net.  I am looking for any resources that might help with getting the project up and running.  Has anyone been able to do this with CI3?

**Comment by jakubmisek** [comment]
I remember we made Code Igniter running, that day with just minor issues. I can't find the original repo though.

Using the boilerplate at https://github.com/iolevel/peachpie-samples/tree/master/web-application, copied CI files into the `website` subfolder and built it. Then the ASP.NET Core pipeline will probably need to be adjusted a bit.

**Comment by rfulcher** [comment]
Thanks for the help with that.  I will give that a try.


[3EAAAAASUVORK5CYII=]      [iiJBAAAAAElFTkSuQmCC]
ROBERT FULCHER

780 S. 2nd St. | Chambersburg, PA 17201
SR. SOLUTIONS ARCHITECT
t (717) 264 - 2434 |  m (717) - 658 - 5100
[kP2wKmHBoLAHib1D2PKgfida8C+hkUCAxAOlzQOoYKZqPAP0KikIGIN0DpC6QohmkkREWMFA+0ZqJAgOnmQUaJdgSygZgYP0B0ho49H5jgSZybGnbgYDFq0Cas6CcQDxJFcVGqIUFAAEGAEFoLmPxjWUfAAAAAElFTkSuQmCC]<https://www.facebook.com/IWAT-250647452810> [GWuRzyHSMSSIpvBwCn90GyGlB4IDtI6x5COcduPkXcC7cRqTnqCE5Kiq2N9r4vvz3YwPciA0TzexAaGvcrwPuErhlAZUqgtkavCIjTjlp2wVzxi7Ff8zmz176H7mbS0BPR+nYAAAAAElFTkSuQmCC] <https://www.instagram.com/iwatllc/>  [RU5oZAAvjFBEzk5YBUotLOgnEAiXfANamEBQIABAOz0PqsydHe4AAAAAElFTkSuQmCC] <https://www.linkedin.com/company/iwat>
***@***.***<https://outlook.office.com/mail/options/mail/messageContent> |  www.iwatllc.com<http://www.iwatllc.com/>



From: Jakub Míšek ***@***.***>
Date: Wednesday, June 14, 2023 at 7:55 AM
To: peachpiecompiler/peachpie ***@***.***>
Cc: Robert Fulcher ***@***.***>, Author ***@***.***>
Subject: Re: [peachpiecompiler/peachpie] Codeigniter 3 (Discussion #1116)

I remember we made Code Igniter running, that day with just minor issues. I can't find the original repo though.

Using the boilerplate at https://github.com/iolevel/peachpie-samples/tree/master/web-application, copied CI files into the website subfolder and built it. Then the ASP.NET Core pipeline will probably need to be adjusted a bit.

**Comment by bfistein** [comment]
Hi @rfulcher - any updates on the progress?

**Comment by rfulcher** [comment]
Benjamin,

Good afternoon.  I really have not had much of a chance to get the code up and running.  Hope to in the next few months.

Thanks


[3EAAAAASUVORK5CYII=]      [iiJBAAAAAElFTkSuQmCC]
ROBERT FULCHER

780 S. 2nd St. | Chambersburg, PA 17201
SR. SOLUTIONS ARCHITECT
t (717) 264 - 2434 |  m (717) - 658 - 5100
[kP2wKmHBoLAHib1D2PKgfida8C+hkUCAxAOlzQOoYKZqPAP0KikIGIN0DpC6QohmkkREWMFA+0ZqJAgOnmQUaJdgSygZgYP0B0ho49H5jgSZybGnbgYDFq0Cas6CcQDxJFcVGqIUFAAEGAEFoLmPxjWUfAAAAAElFTkSuQmCC]<https://www.facebook.com/IWAT-250647452810> [GWuRzyHSMSSIpvBwCn90GyGlB4IDtI6x5COcduPkXcC7cRqTnqCE5Kiq2N9r4vvz3YwPciA0TzexAaGvcrwPuErhlAZUqgtkavCIjTjlp2wVzxi7Ff8zmz176H7mbS0BPR+nYAAAAAElFTkSuQmCC] <https://www.instagram.com/iwatllc/>  [RU5oZAAvjFBEzk5YBUotLOgnEAiXfANamEBQIABAOz0PqsydHe4AAAAAElFTkSuQmCC] <https://www.linkedin.com/company/iwat>
***@***.***<https://outlook.office.com/mail/options/mail/messageContent> |  www.iwatllc.com<http://www.iwatllc.com/>



From: Benjamin Fistein ***@***.***>
Date: Tuesday, August 1, 2023 at 10:02 AM
To: peachpiecompiler/peachpie ***@***.***>
Cc: Robert Fulcher ***@***.***>, Mention ***@***.***>
Subject: Re: [peachpiecompiler/peachpie] Codeigniter 3 (Discussion #1116)

Hi @rfulcher<https://github.com/rfulcher> - any updates on the progress?
